### ISSUE TITLE
ansible-lint 25.9.2 doesn't run on Python 3.14

### ISSUE THREAD

**Issue by ferrarimarco** [ansible/ansible-lint#4822]
##### Summary

When running ansible-lint 25.9.2, I get the following error:

```
RuntimeError: Python 3.14 requires ansible-core version >= 2.20.0, and we found 2.19.3.
```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
#171 2.784 Traceback (most recent call last):
#171 2.785   File "/venvs/ansible-lint/bin/ansible-lint", line 3, in <module>
#171 2.785     from ansiblelint.__main__ import _run_cli_entrypoint
#171 2.785   File "/venvs/ansible-lint/lib/python3.14/site-packages/ansiblelint/__main__.py", line 36, in <module>
#171 2.785     from ansible_compat.prerun import get_cache_dir
#171 2.785   File "/venvs/ansible-lint/lib/python3.14/site-packages/ansible_compat/prerun.py", line 22, in <module>
#171 2.785     raise RuntimeError(msg)
#171 2.785 RuntimeError: Python 3.14 requires ansible-core version >= 2.20.0, and we found 2.19.3.
```

OS: Alpine, running in Docker: python:3.14.0-alpine3.22

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

```console (paste below)
docker run --rm -it --entrypoint /bin/ash python:3.14.0-alpine3.22
pip install ansible-lint
ansible-lint --version
```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Ansible errors out when running `ansible-lint --version`

Please give some details of what is happening. Include a [minimum complete
verifiable example] with:

```paste below
ansible-lint --version
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ferrarimarco** [comment]
Probably related to https://github.com/ansible/ansible-lint/pull/4802 ?

**Comment by alisonlhart** [comment]
Hi @ferrarimarco, this is working as intended. For ansible-lint to work with Python 3.14, ansible-core 2.20 is required. 
There is a release candidate available for ansible-core 2.20, and it is planned for general release on 11-03. 
See here: https://docs.ansible.com/ansible/devel//roadmap/ROADMAP_2_20.html

**Comment by ferrarimarco** [comment]
Ok, thanks!

**Comment by geerlingguy** [comment]
Just wanted to add a 'me too' as I've seen some failures across my CI infrastructure. It looks like [2.20](https://pypi.org/project/ansible-core/2.20.0/) was released yesterday, so it should start trickling out into things like my test container builds. I'll give it a week and let my weekly jobs run again.
